layouts/index.html

```diff
@@ -10,7 +10,7 @@
             {{ $totalpostscount := len (where .Data.Pages "Section" "blog") }}
             {{ $latestpostscount := .Site.Params.latestpostscount | default $totalpostscount }}
             {{ if gt $latestpostscount 0 }}
-              <div class="page-heading">{{ i18n "latestPosts" }}</div>
+              <div class="page-heading">{{ i18n "latestPosts" }}
```

Comment:
should we keep the closing tag on this line as it originally was (so that the `page-heading` class applies only to this line), and instead remove the closing tag on line 26?

Reply:
Yes. There's a misleading indentation here. I'll fix that also.